### DISCUSSION TITLE
App Store Connect API 2.0

### DISCUSSION THREAD

**Discussion by guitaripod** [AvdLee/appstoreconnect-swift-sdk#202] [General]
I'm thinking of contributing with the new API https://developer.apple.com/news/releases/?id=07112022d

Is anyone working on it already? I'd like to avoid duplication of work. Shall we create a GitHub project with tickets?

Cc @AvdLee

**Comment by guitaripod** [comment]
Heh, I went and looked at the releases and saw a release ["Add 2.0",](https://github.com/AvdLee/appstoreconnect-swift-sdk/releases/tag/2.1.0) but when I do `swift build` I get a lot of deprecations. I wonder why that is? 🤔 

Is the API perhaps in beta, and Apple is deprecating things as they go?

**Comment by AvdLee** [comment]
Hi @marcusziade! The latest major 2.0 release already added support for App Store Connect API V2. You should be able to use that :)
